sbi/inference/posteriors/sbi_flow_posterior.py

```diff
@@ -156,6 +175,8 @@ def _sample(self, num_samples: int, context: torch.tensor = None):
         if context is None:
             context = self.context
 
+        print('self.mcmc', self._mcmc_method)
+
         if self._train_with_mcmc:
```

Comment:
I am wondering why this needs to be a class attribute and passed upon construction (as opposed to method argument), given that it is used only once, here in _sample.